src/components/Items/Question/Body.vue

```diff
@@ -203,6 +232,22 @@ export default {
   },
   components: { Textarea },
   methods: {
+    /** handle any additional text that the user may enter after the character count reaches zero
+     * @param {object} event
+     */
+    handleExtraText(event) {
+      // checks if character limit is reached
```

Comment:
this is not needed
The comment below is enough!

Reply:
done